docs/self-managed/operational-guides/configure-flow-control/configure-flow-control.md

```diff
@@ -0,0 +1,311 @@
+---
+id: configure-flow-control
+title: "Configure Flow Control"
+sidebar_label: "Configure Flow Control"
+description: "Learn how to configure Flow Control"
+---
+
+# Configuring Flow Control
+
+When a broker receives a client request, it is written to the **_event
+stream_** first (see section [internal processing](/components/zeebe/technical-concepts/internal-processing.md) for details), and processed later by the stream processor.
+
+If the exporting is slow, if there are many client requests in the stream,
+or processing largely outpaces the exporting rate, it can lead to the backlog
+increasing and the processing latency can grow beyond an acceptable time.
+
+Zeebe 8.6 introduces a new unified flow control mechanism that is able to limit user commands (by default it tries to achieve 200ms response times) and rate limit writes of new records in general.
+
+Limiting the write rate is a new feature that can be used to prevent building up an excessive exporting backlog. There are two ways to limit the write rate, either by setting a static limit or by enabling throttling that dynamically adjust the write rate based on the exporting backlog and rate.
+
+The write rate consists of all new records from processing results, user commands, inter-partition messages, and scheduled tasks.
+
+This feature allows to safeguard that if exporting slows down, writing of new
+records becomes slower too. For user commands this will show up as
+increased backpressure, and observed increase latency and throughput. Internal
+processing will slow down because the stream processor needs to wait longer for writing of processing results to
+complete.
+
+:::note
+The limit and inflight count are calculated per partition.
+:::
+
+```yaml
+zeebe:
+  broker:
+    flowControl:
+      write:
+        enabled: false
+        rampUp: 0
+        limit: 1000
+        throttling:
+          enabled: false
+          acceptableBacklog: 100000
+          minimumLimit: 100
+          resolution: 15s
+```
+
+The zeebe engine can use a static rate limit to restrict the number of
+records that can be written per second, and it can use a dynamic throttling
+which changes the write rate based on the exporting backlog.
+
+The default values can be found in the [Zeebe broker standalone
+configuration template](https://github.com/camunda/camunda/blob/main/dist/src/main/config/broker.standalone.yaml.template) or in the [Zeebe broker configuration template](https://github.com/camunda/camunda/blob/main/dist/src/main/config/broker.yaml.template) in
+the `# flowControl` section.
+
+The `rampUp` value refers to the warmup period or how long it takes to go
+for the current write rate to the one set in the limit (assuming the rate
+higher than the limit). This value is given in seconds and can't be set
+null nor negative.
```

Comment:
:x: ramp up is a time where we slowly increase the effective limit until we reach the configured limit. Basically a way to start with a lower limit while the cluster warms up.

Reply:
Here I corrected the mistake and still left the following phrase, that explains what are the allowed values, not sure if too much information.